paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodIcon.kt

```diff
@@ -27,6 +29,7 @@ internal fun PaymentMethodIcon(
     iconUrl: String?,
     imageLoader: StripeImageLoader,
     iconRequiresTinting: Boolean,
+    iconSize: Dp? = null,
```

Comment:
I'm convinced this is leaking some implementation details to the callers. Why do we need another size here? Can we make it extend to the max? Or what's the purpose of this?

Reply:
yeah, we can make it extend to the max. I made that change. As a result, I also needed to update the horizontal mode icons to set icon `size` instead of just `height` because otherwise it's possible for the icons to take up much more width than they actually need.